cmake/configs/default.cmake

```diff
@@ -51,6 +51,9 @@ hunter_config(CppNetlibUri VERSION 1.0.4-hunter)
 hunter_config(CsvParserCPlusPlus VERSION 1.0.1)
 hunter_config(Eigen VERSION 3.2.4-p0)
 hunter_config(Expat VERSION 2.1.1)
+if(MSVC)
+  hunter_config(getopt VERSION 1.0.0-p0)
```

Comment:
I dont know, I went back and forth. First coded it under wingetopt then changed it to getopt as there is nothing about it that makes it windows only. You could conceivably conpile it for linux or mac but there is no need.